### ISSUE TITLE
Playwright Reporters are entry points

### ISSUE THREAD

**Issue by phiresky** [webpro/knip#423]
In playwright, "custom reporters" can be added to the config file like this:

```ts
	workers: process.env.CI ? 1 : undefined,
	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
	reporter: [
		// JUnit reporter for JUnit format report
		["junit", { outputFile: Configuration.reportName }],
		// Custom reporter for an additional logic
		["./custom-reporter.ts"],
	],
```

In this example, the file "custom-reporter.ts" should be seen by knip as an entry point. Currently, the playwright plugin (https://github.com/webpro/knip/blob/main/packages/knip/src/plugins/playwright/index.ts) only seems to detect the `projects:` as entry points, not the reporters.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.11.0. See [Release 3.11.0](https://github.com/webpro/knip/releases/tag/3.11.0) for release notes._

**Comment by webpro** [comment]
Thanks for _reporting_ this, @phiresky! Should be good now.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v4.0.0. See [Release 4.0.0](https://github.com/webpro/knip/releases/tag/4.0.0) for release notes._
